syntax = "proto3";

option java_multiple_files = true;
option java_package = "ru.itis.std";

package std;

service StdCalculator {
  rpc calculate_std(stream Number) returns (Number) {}
}

message Number {
  double value = 1;
}

